server/api/index.ts

```diff
@@ -4,10 +4,12 @@ import { ReadWriteAPIController } from "./controllers/read-write-api-controller"
 import express from "express";
 import { ExamBankController } from "./controllers/exam-bank-controller";
 import { StudentMiddleware } from "../auth/adfs";
+import { CartoonsController } from "./controllers/cartoons-controller";
 
 const router = express.Router();
 
 ExamBankController.rewriteFile();
+CartoonsController.rewriteFile();
```

Comment:
does this need to happen on startup? just curious

Reply:
I think it's generally safe to. For now especially, since we don't yet have the backend uploading page implemented, this is the easiest option. Once the CMS page for cartoons uploads is developed, I think we can look into only rebuilding these files after uploads and deletes.